### ISSUE TITLE
PHP Compile Error updating to 2.0.2

### ISSUE THREAD

**Issue by AndrewMac** [verbb/comments#258]
### Describe the bug

I have attempted to update to Comments 2.0.2 on my local site, but when I run the "craft up" command I get the following error:

```
> @php craft install/check && php craft up || return 0
PHP Compile Error 'yii\base\ErrorException' with message 'Cannot use Throwable as Throwable because the name is already in use'

in /var/www/html/vendor/verbb/comments/src/services/Comments.php:38

Stack trace:
#0 [internal function]: yii\base\ErrorHandler->handleFatalError()
#1 {main}
```

### Steps to reproduce

1. Use composer to update to Comments 2.0.2
2. Run "php craft up"

### Craft CMS version

4.2.4 Pro

### Plugin version

Update 2.0.1 to 2.0.2

### Multi-site?

Yes

**Comment by engram-design** [comment]
Sorry about this, still trying to get used to this new Git merge tool. Fixed in [2.0.2.1](https://github.com/verbb/comments/releases/tag/2.0.2.1)

**Comment by ghost** [comment]
Thanks so much, once again. I'm always so impressed by how quick you are to get to these things! :-)
